types/node/v4/index.d.ts

```diff
@@ -1,6 +1,8 @@
 // Type definitions for Node.js 4.2
 // Project: http://nodejs.org/
-// Definitions by: Microsoft TypeScript <http://typescriptlang.org>, DefinitelyTyped <https://github.com/DefinitelyTyped/DefinitelyTyped>
+// Definitions by: Microsoft TypeScript <http://typescriptlang.org>,
+//                 DefinitelyTyped <https://github.com/DefinitelyTyped/DefinitelyTyped>
+//                 Marc Ghorayeb <https://github.com/marcghorayeb>
```

Comment:
The `Definitions by:` list is used to determine who gets notified when changes are made to this file. The first two "authors" here can't receive notifications though. It would be good to get some people who would actually respond to new `@types/node` PRs.